image_client.py

```diff
@@ -114,6 +114,8 @@ def upload_to_image_server(self, full_path, redacted, collection, original_path=
         r = requests.post(url, files=files, data=data)
         if r.status_code != 200:
             print(f"Image upload aborted: {r.status_code}:{r.text}")
+            add_imagepath_to_html(local_filename, barcode=remove_non_numerics(os.path.basename(local_filename)),
```

Comment:
adds image path to html file, each time it either passes or fails based on the r.status_code